src/timeseriesinsights/azext_timeseriesinsights/_params.py

```diff
@@ -0,0 +1,88 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+# pylint: disable=line-too-long
+# pylint: disable=too-many-lines
+# pylint: disable=too-many-statements
+
+from azure.cli.core.commands.parameters import (
+    name_type,
+    tags_type,
+    get_enum_type,
+    resource_group_name_type,
+    get_location_type
+)
+from azure.cli.core.commands.validators import get_default_location_from_resource_group
+
+from knack.arguments import CLIArgumentType
+from ._validators import iso_8601_timespace
+
+environment_name_type = CLIArgumentType(id_part='name', help='The name of the Time Series Insights environment associated with the specified resource group.')
+
+
+def load_arguments(self, _):
+
+    with self.argument_context('timeseriesinsights operation list') as c:
+        pass
+
+    with self.argument_context('timeseriesinsights') as c:
+        c.argument('location', get_location_type(self.cli_ctx), validator=get_default_location_from_resource_group)
+        c.argument('tags', tags_type)
+
+    # region environment
+    with self.argument_context('timeseriesinsights environment') as c:
+        from .vendored_sdks.timeseriesinsights.models import SkuName
+        c.argument('environment_name', arg_type=name_type, id_part='name', help='The name of the Time Series Insights environment associated with the specified resource group.')
+        c.argument('location', arg_type=get_location_type(self.cli_ctx))
```

Comment:
Can we just the `location` argument defined in parent group?

Reply:
✔